coalib/results/result_actions/OpenEditorAction.py

```diff
@@ -29,12 +30,16 @@ def is_applicable(result: Result, original_file_dict, file_diff_dict):
         For being applicable, the result has to point to a number of files
         that have to exist i.e. have not been previously deleted.
         """
+
         if not len(result.affected_code) > 0:
-            return False
+            return 'The result is not associated with any source code.'
 
         filenames = set(src.renamed_file(file_diff_dict)
                         for src in result.affected_code)
-        return all(exists(filename) for filename in filenames)
+        if not all(exists(filename) for filename in filenames):
+            return ("The result is associated with source code that doesn't"
+                    'seem to exist.')
```

Comment:
changing the message isn't the solution of duplication, merging them is. I guess for now we can accept the duplication, we have an issue for removing one of the routines (and I think we shuold.)